api/v1alpha1/zync_types.go

```diff
@@ -300,6 +320,35 @@ func (zrs *ZyncRailsSpec) Default() {
 	zrs.MaxThreads = intOrDefault(zrs.MaxThreads, pointer.Int32(zyncDefaultConfigRailsMaxThreads))
 }
 
+// ZyncRailsConsoleSpec configures the App component of System
```

Comment:
wrong comment?

Reply:
fixed in https://github.com/3scale-ops/saas-operator/pull/263/commits/f7909daed91e94d2e466ff1371f807cad17fc5d2